resources/quality/dagoma/dagoma_discoeasy200_pla_fine.inst.cfg

```diff
@@ -0,0 +1,14 @@
+[general]
+version = 3
+name = Fine
+definition = dagoma_discoeasy200
+
+[metadata]
+setting_version = 4
+type = quality
+quality_type = high
```

Comment:
I think `quality_type` high correspond with `layer_height` 0.06. Shouldn't be `normal` here?

Reply:
Fixed in 139ac5c